website/addons/box/static/boxUserConfig.js

```diff
@@ -1,123 +0,0 @@
-            var data = response.result;
-            self.userHasAuth(data.userHasAuth);
-            self.boxName(data.boxName);
-            self.urls(data.urls);
-            self.loaded(true);
-            self.validCredentials(data.validCredentials);
-            self.nNodesAuthorized = data.nNodesAuthorized;
-            if (!self.validCredentials()) {
-                self.changeMessage('Could not retrieve Box settings at ' +
-                    'this time. The Box addon credentials may no longer be valid.' +
-                    ' Try deauthorizing and reauthorizing Box.',
-                    'text-warning');
-            } else if (self.userHasAuth() && self.nNodesAuthorized === 0) {
-                self.changeMessage('Add-on successfully authorized. To link this add-on to an OSF project, ' +
-                    'go to the settings page of the project, enable Box, and choose content to connect.',
-                    'text-success');
-            }
-        },
-        error: function(xhr, textStatus, error){
-            self.changeMessage('Could not retrieve settings. Please refresh the page or ' +
-                'contact <a href="mailto: support@osf.io">support@osf.io</a> if the ' +
-                'problem persists.', 'text-warning');
-            Raven.captureMessage('Could not GET Box settings', {
-                url: url,
-                textStatus: textStatus,
-                error: error
-            });
-        }
-    });
-    // Flashed messages
-    self.message = ko.observable('');
-    self.messageClass = ko.observable('text-info');
-
-    /** Send DELETE request to deauthorize Box */
-    function sendDeauth() {
-        return $.ajax({
-            url: self.urls().delete,
-            type: 'DELETE',
-            success: function() {
-                // Page must be refreshed to remove the list of authorized nodes
-                window.location.reload();
-            },
-            error: function(textStatus, error) {
-                self.changeMessage(language.deauthError, 'text-danger');
-                Raven.captureMessage('Could not deauthorize Box.', {
-                    url: url,
-                    textStatus: textStatus,
-                    error: error
-                });
-            }
-        });
-    }
-
-    /** Change the flashed status message */
-    self.changeMessage = function(text, css, timeout) {
-        self.message(text);
-        var cssClass = css || 'text-info';
-        self.messageClass(cssClass);
-        if (timeout) {
-            // Reset message after timeout period
-            setTimeout(function() {
-                self.message('');
-                self.messageClass('text-info');
-            }, timeout);
-        }
-    };
-
-    /** Pop up confirm dialog for deleting user's access token. */
-    self.deleteKey = function() {
-        bootbox.confirm({
-            title: 'Disconnect Box Account?',
-            message: language.confirmDeauth,
```

Comment:
Please remove these auth messages since they are no longer being used: 
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/osfLanguage.js#L62-#L68